### PR TITLE
Issue 254: Empty Links in Hompage Blogs Stories widget.

### PR DESCRIPTION
**Pull request by tatwell**
Trello Story: https://trello.com/c/JlStwNlL

Issue #254.

### CODE REVIEW

**Comment by mandythomas** [comment]
the url for the blog link for Attallah College is the old one. It's now /education not /ces.

File: .cascade-code/_config/blogs.vm [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#*
```
Comment (tatwell) [inline .cascade-code/_config/blogs.vm]: Notice these new `_config` files.

File: .cascade-code/_config/blogs.vm [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * #set ( $blogTitle = $blogParams[0] )
+ * #set ( $blogCss = $blogParams[1] )
+ *#
+
+## blog id => [ blog title, css theme class ]
+#set ( $BLOGS_CONFIG = {
+  'academics' : [ 'Higher Ed and Technology: Academics at Chapman', '' ],
+  'alumni' : [ 'Chapman Alumni', '' ],
+  'business' : [ 'business', 'Argyros School of Business and Economics', '' ],
+  'education' : [ 'education', 'Attallah College of Educational Studies', '' ],
```
Comment (kaylaziegler) [inline .cascade-code/_config/blogs.vm]: Why do some of these keys have 3 values in the array while others have 2? Also, the order seems wrong for some, like `education` and `business` should be swapped with the second element? Based on the comment at the top of the file: 
```
#set ( $blogTitle = $blogParams[0] )
#set ( $blogCss = $blogParams[1] )
```

Comment (tatwell) [reply .cascade-code/_config/blogs.vm]: Yes, this was a mistake leading to some of the buggy behavior Mandy noticed. Good eye.

**Comment by tatwell** [comment]
That should be the last of the changes for this story.

Call for final review. I'm planning to release tomorrow morning.

**Pull request merged**